extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -453,6 +455,13 @@ private function formatTotalEditorsNumber( $editors ) {
 		return $editors;
 	}
 
+	private function applyBadges( $users ) {
+		return array_map( function( $user ) {
+			$user[ 'badge' ] = $this->getUserBadgeMarkup( User::newFromId( $user[ 'userId' ] )->getEffectiveGroups() );
```

Comment:
I know that "This shouldn't happen" yet this will throw an error if the User object is not created. We don't check for userId presence either.